packages/mermaid/src/utils.ts

```diff
@@ -764,6 +764,9 @@ export const calculateTextDimensions: (
       let cheight = 0;
       const dim = { width: 0, height: 0, lineHeight: 0 };
       for (const line of lines) {
+        if (!line) {
+          continue;
+        }
```

Comment:
Although this fixes the symptom, it doesn't solve the root cause of the issue. 
If we add `<br/><br/><br/>` it should render three empty lines, now, it seems like it won't render anything. 

Maybe, if we replace the `continue` with `line = ' ';`, it might work?

Reply:
Makes sense! I end up with zero-width spaces as they don't occupy horizontal space and won't be collapsed inside DOM elements.